react-client/src/components/Home/Renderers.tsx

```diff
@@ -98,30 +148,38 @@ const Renderers = (
   }
 
   const renderersCards = renderers.map((renderer: Renderer) => allowed == renderer.isAllowed && (
-    <Grid.Col span={12} xs={6} key={renderer.id}>
+    <Grid.Col span={{ base: 12, xs: 6 }} key={renderer.id}>
       <Card shadow='sm' p='lg' radius='md' withBorder>
         <Card.Section withBorder inheritPadding py='xs'>
-          <Group position='apart'>
-            <Text weight={500} color={getNameColor(renderer)}>{renderer.name}</Text>
+          <Group justify='space-between'>
+            <Text fw={500} c={getNameColor(renderer)}>{renderer.name}</Text>
             <Menu withinPortal position='bottom-end' shadow='sm'>
               <Menu.Target>
                 <ActionIcon>
                   <Dots size={16} />
                 </ActionIcon>
               </Menu.Target>
               <Menu.Dropdown>
-                <Menu.Item icon={<ListDetails size={14} />} onClick={() => setAskInfos(renderer.id)}>{i18n.get['Info']}</Menu.Item>
+                <Menu.Item leftSection={<ListDetails size={14} />} onClick={() => setAskInfos(renderer.id)}>{i18n.get['Info']}</Menu.Item>
                 {canModify && (<>
-                  <Menu.Item icon={<Settings size={14} />} color='red' disabled={true /* not implemented yet */}>{i18n.get['Settings']}</Menu.Item>
-                  {!renderer.isAllowed && renderer.uuid && (
-                    <Menu.Item icon={<DevicesPc size={14} />} onClick={() => setAllowed(renderer.uuid, true)} color='green'>{i18n.get['Allow']}</Menu.Item>
+                  <Menu.Item leftSection={<Settings size={14} />} color='red' disabled={true /* not implemented yet */}>{i18n.get['Settings']}</Menu.Item>
```

Comment:
I know this is existing code but is there a plan for this? Does it need an issue?